MAINTAINERS.md

```diff
@@ -1,10 +1,21 @@
-Maintainers of this repository with their focus areas:
+@brian-brazil is the main/default maintainer.
+
+* cmd
+  * promtool: @simonpasquier
+* discovery
+  * k8s: @brancz
+* documentation
+  * prometheus-mixin: @beorn7
+* storage
+  * remote: @csmarchbanks, @cstyan
+* tsdb: @codesome, @krasi-georgiev
```

Comment:
Super tiny nit: isn't TSDB a part of storage.

Reply:
No, they're two different directories. Storage has all the interfaces (plus remote storage).

Reply:
Ups, it's directory-based indeed, make sense.